pkg/v1/tkg/test/config/aws.yaml

```diff
@@ -19,3 +19,5 @@ tkg_config_variables:
   AWS_PRIVATE_SUBNET_ID_2: subnet-016eb6a45203c010e
 management_cluster_options:
   plan: dev
+workload_cluster_options:
```

Comment:
You can add a testcase in https://github.com/vmware-tanzu/tanzu-framework/tree/main/pkg/v1/tkg/test/tkgctl/aws_cc for customclusterbootstrap so that the test is run as part of the pipeline.

i.e. add 
```
var _ = Describe("Functional tests for aws (clusterclass) - Antrea", func() {
	E2ECommonSpec(context.TODO(), func() E2ECommonSpecInput {
		return E2ECommonSpecInput{
			E2EConfig:       e2eConfig,
			ArtifactsFolder: artifactsFolder,
			Cni:             "antrea",
			Plan:            "dev",
			Namespace:       "tkg-system",
			CustomClusterBootstrapFile: <filepath>,
			OtherConfigs:    map[string]string{"clusterclass": "true"},
		}
	})
})
```

Reply:
@ggpaue Please make  sure to update the description above, as ```aws_cc_antrea_test.go``` already uses this description

Reply:
added test case for custom clusterbootstrap